dcos/dcos-ui
# DCOS-12400: highlight tabs with errors

## Testing
1. Go to the create service form
2. Enter invalid data

Error badges should appear in the tabs, and the sum of the numbers in the badges should equal the sum of the number of errors in the error alert banner.

## Trade-offs



## Dependencies



## Screenshots
![Screen Shot 2019-04-18 at 6 13 56 PM](https://user-images.githubusercontent.com/2313998/56394764-51033000-6206-11e9-8596-ea4d5ef838e3.png)
